nototools/lint_config.txt

```diff
@@ -11,3 +11,23 @@ disable gpos/missing
 filename like Bengali
 enable script_required except cp 951-952
 
+condition
+vendor Adobe
+
+# Adobe's position is that these (mathematical) characters are intended to be used with CJK
+# in an LTR environment.  For true bidi text (that would naturally trigger bidi) these
+# characters should come from a font designed for bidi.  So there is no need in this font
+# for the mirrored glyphs.
+disable bidi
+
+# The v1.003 fonts align the windows ascent/descent with the hhea values, but not the
+# typographic ascender
+version == 1.003
+disable head/os2/ascender, head/os2/descender
+
+# Adobe says "None of the zero-width glyphs are involved with attachment"
```

Comment:
@roozbehp:

> "There is no GlyphClassDef subtable of GDEF table in the font, while
> there are non-spacing combining characters: 302A..302D, 3099..309A."

I suspect that this is okay, because our tools didn't find a need to create this particular table. None of the zero-width glyphs are involved with attachment, and that is likely why this table is not needed.